### ISSUE TITLE
docs are missing ?

### ISSUE THREAD

**Issue by jfstassen** [iamstevendao/vue-tel-input#257]
Looks like the docs are gone, alongside the demo website ?

**Comment by iamstevendao** [comment]
@jfstassen Thanks for reporting, it should be fixed now.

**Comment by mzavaletavargas** [comment]
```
error /Users/name/Documents/workspace/wsNode/Dashboard-CRM/node_modules/vue-tel-input: Command failed.
Exit code: 1
Command: cd docs && npm ci --prefer-offline
Arguments: 
Directory: /Users/name/Documents/workspace/wsNode/Dashboard-CRM/node_modules/vue-tel-input
Output:
/bin/sh: line 0: cd: docs: No such file or
``` 

Having the same issue with node 12.21 and vue-tel-input 5.3.0

**Comment by iamstevendao** [comment]
#268 @mzavaletavargas it should be fixed in [v5.5.0](https://github.com/iamstevendao/vue-tel-input/releases/tag/v5.5.0)
